--- src/Makefile.am	2009-01-07 10:32:29.000000000 +0100
+++ src/Makefile.am.oden	2009-01-07 10:32:37.000000000 +0100
@@ -8,5 +8,5 @@
 mod_parmguard_la_SOURCES 	= mod_parmguard.c conf.c rules.c check.c conf.h rules.h common.h
 mod_parmguard_la_LDFLAGS	= -lxml2 -module -shared -s -avoid-version
 checkconf_SOURCES		= checkconf.c conf.h rules.h
-checkconf_LDFLAGS		= conf.o rules.o
+checkconf_LDFLAGS		= conf.lo rules.lo
 
